src/elm/Page/Transfer.elm

```diff
@@ -0,0 +1,413 @@
+                ]
+
+        Loaded maybeTransfer state ->
+            case maybeTransfer of
+                Just transfer ->
+                    div []
+                        [ Lazy.lazy viewHeader loggedIn
+                        , div []
+                            [ viewTransfer loggedIn transfer state
+                            , viewDetails loggedIn transfer
+                            ]
+                        ]
+
+                Nothing ->
+                    div [ class "container mx-auto px-4" ]
+                        [ div []
+                            [ text "Could not load the transfer" ]
+                        ]
+
+
+viewHeader : LoggedIn.Model -> Html Msg
+viewHeader ({ shared } as loggedIn) =
+    let
+        t =
+            I18Next.t loggedIn.shared.translations
+    in
+    div [ class "h-16 w-full bg-indigo-500 flex px-4 items-center" ]
+        [ a
+            [ class "items-center flex absolute"
+            , Route.href Route.Dashboard
+            ]
+            [ Icons.back ""
+            , p [ class "text-white text-sm ml-2" ]
+                [ text (I18Next.t shared.translations "back")
+                ]
+            ]
+        , p [ class "text-white mx-auto" ] [ text (t "transfer_result.title") ]
+        ]
+
+
+viewTransfer : LoggedIn.Model -> Transfer -> State -> Html Msg
+viewTransfer loggedIn transfer state =
+    let
+        t =
+            I18Next.t loggedIn.shared.translations
+    in
+    div [ class "flex" ]
+        [ div [ class "w-full bg-green h-50" ]
+            [ div [ class "flex-row" ]
+                [ div [ class "px-4 py-2 m-2" ]
+                    [ h2 [ class "text-center mt-8 font-medium font-sans text-white not-italic" ]
+                        [ text <|
+                            case state of
+                                Transferred ->
+                                    t "transfer_result.transfer_success"
+
+                                Received ->
+                                    t "transfer_result.receive_success"
+                        ]
+                    ]
+                , div [ class "bg-no-repeat bg-auto h-64 ml-32 -mt-5 px-4 py-2 m-2 items-center mt-5 justify-center", style "background-image" "url(/images/transfer-doggo.svg)" ]
+                    []
+                , div [ class "flex sm:justify-center sm:items-center -mt-16 absolute md:w-full md:mx-auto  sm:right-0" ]
+                    [ viewTransferCard loggedIn transfer state
+                    ]
+                ]
+            ]
+        ]
+
+
+viewTransferCard : LoggedIn.Model -> Transfer -> State -> Html Msg
+viewTransferCard loggedIn transfer state =
+    let
+        ipfsUrl =
+            loggedIn.shared.endpoints.ipfs
+    in
+    div [ class "flex flex-row block rounded overflow-x-scroll sm:overflow-hidden bg-gray-100" ]
+        [ div [ class "sm:px-4 sm:py-2 sm:m-2 p-0" ]
+            [ div [ class "h-8 w-8 rounded-full mx-auto mt-5" ]
```

Comment:
we had a few problems with having the avatar display different, lets extract a function that will only render the given avatar! Please put this after `viewTransferCard`:


```elm
viewTransferAvatar : LoggedIn.Model -> TransferUser -> Html Msg
viewTransferAvatar loggedIn user =
    div []
        [ div [ class "w-8 h-8 rounded-full mx-auto" ]
            [ Avatar.view loggedIn.shared.endpoints.ipfs user.avatar "w-10 h-10"
            ]
        , div [ class "px-6 py-4" ]
            [ div [ class "inline-block bg-black rounded px-3 py-1 text-xs uppercase font-medium text-white" ]
                [ if user.account == loggedIn.accountName then
                    text (I18Next.t loggedIn.shared.translations "transfer_result.you")

                  else
                    case user.userName of
                        Just username ->
                            text username

                        Nothing ->
                            Eos.viewName user.account
                ]
            ]
        ]


```



After this this function should look like this:

```elm
viewTransferCard : LoggedIn.Model -> Transfer -> State -> Html Msg
viewTransferCard loggedIn transfer state =
    let
        ipfsUrl =
            loggedIn.shared.endpoints.ipfs

        originUser =
            case state of
                Received ->
                    transfer.to

                Transferred ->
                    transfer.from

        destinationUser =
            case state of
                Received ->
                    transfer.from

                Transferred ->
                    transfer.to
    in
    div [ class "flex flex-row w-full justify-center items-center py-5 rounded bg-gray-100" ]
        [ viewTransferAvatar loggedIn originUser
        , viewAmount loggedIn transfer state
        , viewTransferAvatar loggedIn destinationUser
        ]
```

Much simpler! 😁

Reply:
Also note how now we have the "You" translatable! You'll have to add the translations for it to work properly!

Reply:
Another note on the div in `transferCard`: We now rely on the class `items-center` to align it vertically, instead of using margins!

Reply:
Much simpler!